pkg/operator/ceph/cluster/nodedaemon/ceph-exporter.go

```diff
@@ -0,0 +1,241 @@
+/*
```

Comment:
How about the names exporter.go and exporter_test.go? No need for the ceph prefix